eng/versioning/pom_file_version_scanner.ps1

```diff
@@ -444,8 +444,8 @@ Write-Host "Total run time=$($TotalRunTime)"
 
 if ($script:FoundError)
 {
-    Write-Host "There were errors encountered during execution. Please fix any errors and run the script again."
-    Write-Host "This script can be run locally from the root of the repo. .\eng\pom_file_version_scanner.ps1"
+    Write-Error-With-Color "There were errors encountered during execution. Please fix any errors and run the script again."
+    Write-Error-With-Color "This script can be run locally from the root of the repo. .\eng\pom_file_version_scanner.ps1"
```

Comment:
Could we add line telling user that, it needs to be run in powershell. Also it need certain permission in powershell, I had issues running it yesterday . How to set permission in PS https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-6

Reply:
I can add something to the description at the top of the file for local setup. The images doing the building already have the correct permissions in place.